index.js

```diff
@@ -17,7 +17,8 @@
 'use strict';
 
 var OpaqueSpan = require('./lib/opaque-span.js');
-var Logger = require('./lib/logger.js');
+var CloudCommon = require('@google/cloud-diagnostics-common');
+var Logger = CloudCommon.logger;
```

Comment:
Tracking logger capitalization in common here: https://github.com/matthewloring/cloud-diagnostics-common-nodejs/issues/1